src/app/public/theming/fixtures/theme-class-test.component.html

```diff
@@ -0,0 +1,21 @@
+<section>
+  <div
+    [skyThemeClass]="{
```

Comment:
There are a handful of style issues here:

- There should be spaces between properties and their values (our TSLint rules would catch this in a .ts file):
```
'example-modern': 'modern'
```
- Property values should be alphabetized
- Attributes should be ordered using the rules outlined in the SKY UX Team Style Guide (hyperlink below)
- There should be a space on either side of an operator (our TSLint rules would catch this in a .ts file)
```
'sky-theme-class-test ' + exampleString
```

The SKY UX Team Style Guide is available here: https://docs.blackbaud.com/skyux-team-docs/style